docs/agents/sematext-agent/releasenotes.md

```diff
@@ -39,6 +39,22 @@ NULL
 
 -->
 
+## Version 3.7.0
+
+Date: September 23, 2024
+
+### New Features
+- **Windows Events**: We are happy to share that Sematext Agent now supports collecting Windows Events. You can define which event logs to collect and send to Sematext Cloud, refer to the [configuration documentation](https://sematext.com/docs/integration/configuring-windows-agent/).
```

Comment:
This link is broken right now. https://github.com/sematext/docs/pull/640/files#diff-3e6a67ec8a92f5304ccb67dfab4ba6c92a02eacc2fa6cba44fcefb3d8a9ce846